examples/gpt/schedule.py

```diff
@@ -168,9 +168,16 @@ def pattern(x: torch.Tensor) -> torch.Tensor:
         if sch.world_size > 1:
             sub_sch["module.FusedQKV_0.fused_linear"].shard("weight", axis=0)
             sub_sch["module.FusedQKV_0.fused_linear"].shard("bias", axis=0)
-            sub_sch["module.FusedQKV_0.fused_linear"].sync(mode="backward")
+            sub_sch["module.FusedQKV_0.fused_linear"].sync(
+                mode="bwd_post", sync_op_or_fn="all_reduce"
+            )
             sub_sch["module.out_proj"].shard("weight", axis=1)
-            sub_sch["module.out_proj"].sync(mode="forward")
+            sub_sch["module.out_proj"].sync(
+                mode="fwd_post", sync_op_or_fn="reduce_scatter", axis=0
+            )
```

Comment:
I thought it should be axis=1? axis=1 is the sequence dimension?

Reply:
Oops, good catch.